docs/reference/planners/ENHSP/main.md

```diff
@@ -15,22 +15,30 @@ Year Published: 2016
 
-ENHSP, which stands for Expressive Numeric Heuristic Planner is a forward heuristic search planner. It supports the following 
+ENHSP, which stands for Expressive Numeric Heuristic Planner is a forward heuristic search planner. It supports PDDL2.1 and discretised PDDL+, in particular: 
 
-1. Classical Planning
-2. Numeric Planning with linear and non-linear expressions
+1. Satisficing and Optimal Simple Numeric Planning (see below)
+2. Satisficing Numeric Planning with linear and non-linear expressions
 3. Planning with discretised autonomous processes and events
 4. Global constraints, which are the analogous of always constraints of PDDL
-5. discrete events
-6. universal and existential quantification in formulas (action precondition, constraints and goals)
-7. negative preconditions
-8. novel heuristics based on landmarks and search techniques for planning with autonomous processes
+5. Good support for general quantifier-free formulas
+6. Universal and existential quantification in formulas (action precondition, constraints and goals). This is hardly experimental.
+7. A variety of heuristics (e.g., hadd, aibr, haddabs, landmarks) and search mechanisms (e.g., wastar, idastar, dfsbnb)
 
 ## Support
 ENHSP requires `:typing` in all domains it runs on, which makes it incompatible with [Eviscerator](https://www.github.com/nergmada/eviscerator). No support table has been generated. 
 
 ## Downloading and Compiling ENHSP
-For more information on how to download, compile and use ENHSP, visit its [GitLab page here](https://gitlab.com/enricos83/ENHSP-Public). 
+For more information on how to download, compile and use ENHSP, visit its [GitLab page here](https://gitlab.com/enricos83/ENHSP-Public). The planner is mantained by Enrico Scala (enricos83@gmail.com for any question).
 
 ## Additional Notes
-ENHSP has support for non-linear continuous effects assuming they are monotonic.
\ No newline at end of file
+- ENHSP grounding is not optimised; this can become a bottleneck when your action schema induce several even trivial groundings.
+- ENHSP does not support durative actions
+- Conditional effects are supported but are highly experimental
+- Optimal planning is only supported for what is called simple numeric planning (including classical planning), i.e., only increase/decrease effects of constants, and only linear numeric conditions. Objective function can be linear combination of numeric fluents. If no objective function is given, the planner minimises the number of actions.
+- ENHSP is designed to experiment on numeric reasoning. If your problem is a purely classical planning problem, it is better to use a classical planner like FD, or even FF.
+
+## Other Relevant Papers ENHSP is built on
+[Heuristics for Numeric Planning via Subgoaling](https://www.ijcai.org/Proceedings/16/Papers/457.pdf) [Scala, E., Haslum, P. and Thiébaux, S.]
```

Comment:
We need to move over to a citation based approach, see how citations are done on the main page, but not worried that this is in the old format